lib/ReactViews/Map/MapNavigation/Items/MyLocation.ts

```diff
@@ -57,6 +57,9 @@ export class MyLocation extends MapNavigationItemController {
         timeout: 5000,
         maximumAge: 0
       };
+      navigator.permissions.query({ name: "geolocation" }).then((result) => {
+        console.log(result);
```

Comment:
Does this log the result of each user location query to the console of the server? Aren't there potentially thousands of concurrent users?

Reply:
forgot to push changes, thanks :D

Reply:
It seems I can't resolve this, but what I commented about seems fixed.